[2025-09-11T06:51:39.908+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-11T06:51:40.037+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_daily_monitoring.check_predictions manual__2025-09-11T06:51:37.649164+00:00 [queued]>
[2025-09-11T06:51:40.084+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_daily_monitoring.check_predictions manual__2025-09-11T06:51:37.649164+00:00 [queued]>
[2025-09-11T06:51:40.086+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-11T06:51:40.144+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_predictions> on 2025-09-11 06:51:37.649164+00:00
[2025-09-11T06:51:40.222+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_daily_monitoring', 'check_predictions', 'manual__2025-09-11T06:51:37.649164+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/monitor_dag.py', '--cfg-path', '/tmp/tmp961xrs0z']
[2025-09-11T06:51:40.248+0000] {standard_task_runner.py:91} INFO - Job 11: Subtask check_predictions
[2025-09-11T06:51:40.259+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=387) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-11T06:51:40.266+0000] {standard_task_runner.py:64} INFO - Started process 392 to run task
[2025-09-11T06:51:40.665+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_daily_monitoring.check_predictions manual__2025-09-11T06:51:37.649164+00:00 [running]> on host 02ae08abed04
[2025-09-11T06:51:41.260+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ops-team' AIRFLOW_CTX_DAG_ID='simple_daily_monitoring' AIRFLOW_CTX_TASK_ID='check_predictions' AIRFLOW_CTX_EXECUTION_DATE='2025-09-11T06:51:37.649164+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-11T06:51:37.649164+00:00'
[2025-09-11T06:51:41.267+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-11T06:51:41.358+0000] {logging_mixin.py:188} INFO - üìà Checking recent predictions...
[2025-09-11T06:51:41.362+0000] {logging_mixin.py:188} INFO - ‚ùå No predictions file found
[2025-09-11T06:51:41.371+0000] {python.py:237} INFO - Done. Returned value was: no_predictions
[2025-09-11T06:51:41.372+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-11T06:51:41.497+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_daily_monitoring, task_id=check_predictions, run_id=manual__2025-09-11T06:51:37.649164+00:00, execution_date=20250911T065137, start_date=20250911T065140, end_date=20250911T065141
[2025-09-11T06:51:42.335+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-11T06:51:42.398+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-11T06:51:42.402+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
